src/model/writer.js

```diff
@@ -574,7 +564,9 @@ export default class Writer {
 		const delta = new RenameDelta();
 		this.batch.addDelta( delta );
 
-		const renameOperation = new RenameOperation( Position.createBefore( element ), element.name, newName, this.model.document.version );
+		const version = element.root.document ? this.model.document.version : null;
```

Comment:
More natural is: `element.root.document ? element.root.document.version : null;`